sessionize/app/src/main/res/layout/item_event.xml

```diff
@@ -50,6 +50,7 @@
                 android:layout_marginEnd="@dimen/live_icon_margin"
                 android:contentDescription="@string/cd_live_now"
                 android:src="@drawable/ic_play_24dp"
+                android:visibility="invisible"
```

Comment:
This is not currently used, making it invisible so we don't have to actively hide it ever time